pkg/buildexpression/parser/parser_test.go

```diff
@@ -0,0 +1,49 @@
+package parser
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/ActiveState/cli/internal/environment"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPaser_Parse(t *testing.T) {
```

Comment:
It's great that you have tests here, but they're only testing a lack of failure. You're not testing that the parser is producing the AST you expect it to.